web/portal/client/src/entities/HolidayDate/Action/Import.tsx

```diff
@@ -113,20 +200,70 @@ const Import: ActionFunctionComponent = (props: MultiSelectActionItemProps) => {
         onClose={handleClose}
         aria-labelledby='alert-dialog-title'
         aria-describedby='alert-dialog-description'
+        className={step === 1 ? '' : 'width'}
       >
         <Box>
           <DialogTitle id='alert-dialog-title'>
-            {_('Import Holiday Dates')}
+            {_('Import Holiday dates')}
```

Comment:
If you change this capital letter, be sure to update all the texts with the same label (line 184 and 188 of this same file) to minimize the number of translatable strings. Choose one (dates or Dates) and update all occurrences.